src/arch/x86/object/vcpu.c

```diff
@@ -28,9 +28,14 @@
 #define VMXON_REGION_SIZE 4096
 
 const vcpu_gp_register_t crExitRegs[] = {
-    VCPU_EAX, VCPU_ECX, VCPU_EDX, VCPU_EBX, VCPU_ESP, VCPU_EBP, VCPU_ESI, VCPU_EDI
+    VCPU_EAX, VCPU_ECX, VCPU_EDX, VCPU_EBX, VCPU_ESP, VCPU_EBP, VCPU_ESI, VCPU_EDI,
```

Comment:
I think keeping EAX-EDI for ia32 and adding RAX-RDI for x86-64 could be less confusing.